R/weather_radars.R

```diff
@@ -0,0 +1,38 @@
+#' Get weather radar data
+#'
+#' The `weather_radars()` function is used to retreive weather radar metadata
+#' from the [OPERA network](https://www.eumetnet.eu/activities/observations-programme/current-activities/opera/).
+#' The retrieved data is returned as a tibble.
+#'
+#' @return A tibble containing weather radar metadata
+#' @export
+#'
+#' @examplesIf interactive()
+#' weather_radars()
+weather_radars <- function() {
+  # Build the url where the JSON file is hosted on Github
+  opera_radar_overview_url <-
+    paste(
+      sep = "/",
+      "https://raw.githubusercontent.com/enram",
+      "aloftdata.eu/refs/heads/main/_data",
+      "OPERA_RADARS_DB.json"
+    )
+
+  # Fetch the JSON file from Github with similar arguments as the other
+  # functions
+  httr2::request(opera_radar_overview_url) |>
+    req_user_agent_getrad() |>
+    httr2::req_retry(
+      max_tries = 15, backoff = \(x) sqrt(x) * 2,
+      is_transient = \(resp) httr2::resp_status(resp) %in% c(429),
+      retry_on_failure = TRUE
```

Comment:
This is resulting in a:

> Error in httr2::req_retry(req_user_agent_getrad(httr2::request(opera_radar_overview_url)),  : 
  unused argument (retry_on_failure = TRUE)
  
I use httr2 1.0.3.9000

Reply:
Works fine in httr2 1.0.7, so maybe we have to define that in DESCRIPTION

Reply:
I've lifted that line strait out of the other get_pvol helpers, the argument means:

> Treat low-level failures as if they are transient errors, and can be retried.

and is set to FALSE by default. I kept it this way to be consistent with the other functions. We could also make changes to allow for a lower version of `httr2`